Sources/RunnerLib/Files Import/DangerFileGenerator.swift

```diff
@@ -10,7 +10,7 @@ public final class DangerFileGenerator {
 
         importsRegex.enumerateMatches(in: content, options: [], range: NSRange(location: 0, length: content.count), using: { result, _, _ in
             // Adjust the result to have the correct range also after dangerContent is modified
-            guard let result = result?.adjustingRanges(offset: dangerContent.count - content.count) else { return }
+            guard let result = result?.adjustingRanges(offset: dangerContent.endIndex.encodedOffset - content.endIndex.encodedOffset) else { return }
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/RunnerLib/Files Import/DangerFileGenerator.swift;
  Line: 13;
-->

- :warning: Line should be 120 characters or less: currently 149 characters (`line_length`)